app/parsers/bulkrax/csv_parser_decorator.rb

```diff
@@ -16,7 +17,7 @@ def valid_import?
 
       file_paths.is_a?(Array)
     rescue StandardError => e
-      status_info(e)
```

Comment:
this method was deprecated in Bulkrax v6.0.0. Because of this, the imports would fail silently if there's an issue.